engine/src/main/java/org/terasology/engine/module/StandardModuleExtension.java

```diff
@@ -56,6 +61,31 @@ public static boolean isGameplayModule(Module module) {
         return isGameplay != null && isGameplay;
     }
 
+    public static boolean isAssetModule(Module module) {
+        Boolean isAssetplay = module.getMetadata().getExtension(IS_ASSET.getKey(), Boolean.class);
```

Comment:
Personally, since it's used in multiple methods, I would have extracted the code in common between this and the other boolean getters to a separate private method, i.e. something like
```
private static boolean getBooleanExtension(Module module, StandardModuleExtension ext) {
    Boolean result = module.getMetadata().getExtension(ext.getKey(), Boolean.class);
    return result != null && result;
}
```
and then rewrite this and the other getters as
```
public static boolean isAssetModule(Module module) {
    return getBooleanExtension(module, IS_ASSET);
}
```
but I see that the similar accessor methods before your changes were like yours, so not a big deal probably.

Oh and by the way, you probably meant `isAsset` instead of `isAssetplay` here.